src/twisted/application/runner/_exit.py

```diff
@@ -66,72 +40,60 @@ class Status:  # type: ignore[no-redef]
         EX_CONFIG = EX__BASE + 14
 
 
-class ExitStatus(Values):
```

Comment:
Used an `IntEnum` here instead of constantly because type support is far cleaner and we don't need anything from `constantly` here.

Reply:
Note: compatibility should be OK since this is not yet public API.